### PR TITLE
Mongo integ

### PR DESCRIPTION
**Pull request by bfishbaum**
Added a mongoUtil.js file that loads the mongo database and exposes it to whichever express files need to use it

### CODE REVIEW

File: express/app.js [hunk 1]
```diff
@@ -20,4 +24,29 @@ app.get('/message', (req, res) => {
 	})
 })
 
-app.listen(port, () => console.log("listening"));
+app.use('/api',apiRouter)
+
+app.get("/db/add", function(req, res, next) {
+	database.collection(TICname).updateOne({"name":"CounterDoc"}, {$set:{"counter": counter}}, {upsert: true})
```
Comment (machang19) [inline express/app.js]: These two methods get and see just check writing and reading from the database correct?

File: express/mongoUtil.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+		started = true;
+	})
+}
+
+const getDatabase = () => {
+	return database
+}
+
+/*
+dbRouter.get("/add", function(req, res, next) {
```
Comment (machang19) [inline express/mongoUtil.js]: Remove dead code

File: express/mongoUtil.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+var counter = 0
+var started = false
+
+const mongoOptions = {
+		useNewUrlParser: true, 
+		reconnectTries: 100, 
+		reconnectInterval: 1000,
+		autoReconnect: true
+}
+const url = "mongodb://mongodb:27017/";
```
Comment (machang19) [inline express/mongoUtil.js]: Can we interact with database at this url when running?

**Comment by machang19** [comment]
+1

File: express/mongoUtil.js (hunk 3 above)
Comment (bfishbaum) [reply express/mongoUtil.js]: All my changes worked in the production Docker build, so I believe so. The first 'mongodb' refers to how the data over that url will be transferred, and the second is the name of the docker container the mongodb is built in.

**Pull request merged**